source/developers/transform-data-with-object-lambda.rst

```diff
@@ -0,0 +1,242 @@
+    * Running on http://127.0.0.1:5000
+   Press CTRL+C to quit
+
+Once the handler is running, start MinIO with the :envvar:`MINIO_LAMBDA_WEBHOOK_ENABLE` and :envvar:`MINIO_LAMBDA_WEBHOOK_ENDPOINT` environment variables to register the function with MinIO.
+To identify the specific Object Lambda handler, append the name of the function to the name of the environment variable:
+
+.. code-block:: shell
+   :class: copyable
+
+   MINIO_LAMBDA_WEBHOOK_ENABLE_myfunction=on MINIO_LAMBDA_WEBHOOK_ENDPOINT_myfunction=http://localhost:5000 minio server /data &
+
+Replace ``myfunction`` with the name of the handler function and ``/data`` with the location of the MinIO directory for your local deployment. 
+The output resembles the following:
+
+.. code-block:: shell
+
+   MinIO Object Storage Server
+   Copyright: 2015-2023 MinIO, Inc.
+   License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
+   Version: RELEASE.2023-03-24T21-41-23Z (go1.19.7 linux/arm64)
+   
+   Status:         1 Online, 0 Offline. 
+   API: http://192.168.64.21:9000  http://127.0.0.1:9000       
+   RootUser: minioadmin 
+   RootPass: minioadmin 
+   Object Lambda ARNs: arn:minio:s3-object-lambda::myfunction:webhook 
+
+
+Test the Handler Function
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To test the Lambda handler function, first create an object to transform.
+Then invoke the handler, in this case from a test function written in Go.
+
+#. Create a bucket and object for the handler to transform.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc alias set myminio/ http://localhost:9000 minioadmin minioadmin
+      mc mb myminio/myfunctionbucket
+      cat > testobject << EOF
+      Hello, World!
+      EOF
+      mc cp testobject myminio/myfunctionbucket/
+
+#. Invoke the Handler
+
+   The following Go code uses the :doc:`The MinIO Go SDK </developers/go/minio-go>` to generate a presigned URL and print the transformed object to ``stdout``.
+
+   .. code-block:: go
+      :class: copyable
+
+      package main
+
+      import (
+         "context"
+         "log"
+         "net/url"
+         "time"
+         "fmt"
+
+         "github.com/minio/minio-go/v7"
+         "github.com/minio/minio-go/v7/pkg/credentials"
+      )
+
+      func main() {
+         s3Client, err := minio.New("localhost:9000", &minio.Options{
+            Creds:  credentials.NewStaticV4("minioadmin", "minioadmin", ""),
+            Secure: false,
+         })
+         if err != nil {
+            log.Fatalln(err)
+         }
+
+         // Set lambda function target via `lambdaArn`
+         reqParams := make(url.Values)
+         reqParams.Set("lambdaArn", "arn:minio:s3-object-lambda::myfunction:webhook")
+
+         // Generate presigned GET url with lambda function
+         presignedURL, err := s3Client.PresignedGetObject(context.Background(), "myfunctionbucket", "testobject", time.Duration(1000)*time.Second, reqParams)
```

Comment:
Oops need some text noting where to put your own function, bucket, and object names.